### PR TITLE
Update NodeJS version on TravisCI to 6.7.0

### PR DESCRIPTION
**Pull request by DanReyLop**
As of #590, we tested the client using nodejs `6.7.0`, but we didn't update the version running in our CI. This PR bumps that version.

This doesn't need to be tested. If the TravisCI passes, it's good to go.

@nabsul @jeffstieler @robobot3000 @allendav

### CODE REVIEW

**Comment by nabsul** [comment]
Travis CI seems to be failing....

**Comment by DanReyLop** [comment]
So, here is a recap:

CI was failing because `node-sass` is compatible with node 6 only from version `3.7.0` and up. The version we had in our `shrinkwrap` was `3.4.2`. We were locking a version of `wp-calypso` from mid-June, before *they* moved to node `6.7.0`. As a quick fix, I edited manually the shrinkwrap, and it works (it failed in one machine, but that's probably Travis fault).

Then, I fixed `shrinkwrap`, maybe once and for all. I made a new `npm` task: `npm run update-deps`. Look at what it does, it takes a while. Maybe it's overkill but I'm not taking any chances.

Hopefully we won't need to nuke and regenerate the `shrinkwrap` file any more. According to the NPM documentation, using `npm ... --save` to install/uninstall/update dependencies (as oposed to manually editing `package.json`) will update both `package.json` **and** `npm-shrinkwrap.json`. This was buggy until a while ago, so I recommend installing the latest version of NPM (`npm install npm --global`).

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "test": "karma start --single-run --no-auto-watch && npm run eslint",
     "test-watch": "karma start --auto-watch --no-single-run",
     "release": "node tasks/release.js",
-    "i18n": "npm run dist && node tasks/i18n"
+    "i18n": "npm run dist && node tasks/i18n",
+	"update-deps": "rm npm-shrinkwrap.json && rm -rf node_modules && npm install --no-optional && npm install --no-optional && npm prune && npm shrinkwrap --dev"
```
Comment (DanReyLop) [inline package.json]: Yeah, this is that `wp-calypso` does. I really don't want to learn why do they have to `npm install` twice, I prefer to live in ignorance.

Note that this is only needed to regenerate the `shrinkwrap` file, you **shouldn't** need to do this almost ever.

File: client/config.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+export default ( key ) => {
+	if ( key in data ) {
+		return data[ key ];
+	}
+	throw new Error( 'config key `' + key + '` does not exist' );
+};
+
+export const isEnabled = () => false;
+export const anyEnabled = () => false;
```
Comment (jeffstieler) [inline client/config.js]: Can you explain the need for this file? Which Calypso components (that we're using) require this?

**Pull request merged**